src/app/record/components/work/work.component.html

```diff
@@ -33,15 +33,18 @@
       >
         <ng-container
           *ngIf="
-            contributor?.creditName && contributor?.creditName?.content !== ''
+            contributor?.creditName &&
+            contributor?.creditName?.content !== '' &&
+            contributor?.creditName?.content !== privateName
           "
         >
           {{ contributor?.creditName?.content + (!last ? '; ' : '') }}
         </ng-container>
         <ng-container
           *ngIf="
             (!contributor?.creditName ||
-              contributor?.creditName?.content === '') &&
+              contributor?.creditName?.content === '' ||
```

Comment:
Looking at the backend logic 
https://github.com/ORCID/ORCID-Source/pull/6533/files#diff-95458e03beba5286beb751702a1fd06f126e97d1f1c4b52bf4a30d7d9c530d73R205

We don't need to check `contributor?.creditName?.content === ''` since it will never be true

Reply:
(also for line 38)

Reply:
@leomendoza123 Great catch. I will remove the unnecessary validation